pandas/core/config_init.py

```diff
@@ -307,6 +307,48 @@ def mpl_style_cb(key):
     return val
 
 
+def table_schema_cb(key):
+    # Having _ipython_display_ defined messes with the return value
+    # from cells, so the Out[x] dicitonary breaks.
```

Comment:
don't write the full method here
instead inside the cb import a method a call it